### PR TITLE
Refactor DAGs to better conform to Astronomer style opinions

### PR DESCRIPTION
**Pull request by denimalpaca**

### CODE REVIEW

File: dags/dbt_advanced.py [hunk 1]
```diff
@@ -1,89 +1,72 @@
-import datetime
 import json
 from airflow import DAG
 from airflow.operators.bash_operator import BashOperator
 from airflow.utils.dates import datetime
```
Comment (josh-fell) [inline dags/dbt_advanced.py]: We shouldn't be importing from `datetime` from `airflow.utils.dates`. Ideally we should really be importing `pendulum.datetime` for `start_date`. Something on my backlog to transition our certified DAGs to at some point (see [this documentation](https://airflow.apache.org/docs/apache-airflow/stable/timezone.html#interpretation-of-naive-datetime-objects)). It's not a big deal (`datetime.datetime` vs. `pendulum.datetime`) unless folks want to use timezone-aware DAGs which _require_ `pendulum.datetime`. Just a good habit now.

File: dags/dbt_advanced.py [hunk 2]
```diff
@@ -1,89 +1,72 @@
...
-    'email': ['noreply@astronomer.io'],
-    'email_on_failure': False
-}
-
-dag = DAG(
+with DAG(
     'dbt_advanced_dag',
-    default_args=default_args,
+    start_date=datetime(2020, 12, 23),
     description='A dbt wrapper for airflow',
```
Comment (josh-fell) [inline dags/dbt_advanced.py]:
```suggestion
    description='A dbt wrapper for Airflow.',
```

**Review by josh-fell — CHANGES_REQUESTED** [review]
Thanks for sprucing these up!

Small comment on some new developments/best practices being used in OSS example DAGs that we should carry over re: using `pendulum.datetime` for `start_date`.

File: dags/dbt_advanced.py (hunk 1 above)
Comment (denimalpaca) [reply dags/dbt_advanced.py]: Noted and done.

File: requirements.txt [hunk 3]
```diff
@@ -1 +1,5 @@
 dbt==0.21.1
+tap-github==1.9.2
+target-csv==0.3.0
+jsonschema==3.2.0
```
Comment (josh-fell) [inline requirements.txt]:
```suggestion
```
Not needed since `elt.py` is no longer a part of the repo.

**Review by josh-fell — CHANGES_REQUESTED** [review]

**Review by josh-fell — APPROVED** [review]

**Pull request merged**